app/helpers/application_helper.rb

```diff
@@ -1,18 +1,34 @@
 module ApplicationHelper
   def signin_path(provider)
-    redir_path = {origin: request.env['PATH_INFO']}.to_query
+    redir_path = {origin: request.env["PATH_INFO"]}.to_query
     "/auth/#{provider.to_s}?#{redir_path}"
   end
 
   def create_event_or_login(name, classes)
     if current_user
-      content_tag(:a, name, href:events_new_path, class: classes*' ' )
+      content_tag(:a, name, href:events_new_path, class: classes*" " )
```

Comment:
Space missing after colon.<br>Surrounding space missing for operator `*`.<br>Space inside parentheses detected.